src/gatsby-plugin-theme-ui/2020.js

```diff
@@ -0,0 +1,19 @@
+import baseTheme from 'gatsby-plugin-theme-ui'
+
+export default {
+    ...baseTheme,
+ colors: {
+   ...baseTheme.colors,
+   darkBlack: '#1b1c1e',
+   black: '#1f2123',
+   darkBlack: '#1b1c1e',
+   lightBlack: '#1f2123',
+   grayHeader: '#555e68',
+   grayParagraph: '#888b9b',
+   twentyTwentyBlue: '#395DED',
```

Comment:
shouldn't we follow the same conventions? e.g. `twentyTwentyBlue` = `primary`

mapping the functional names to the colors will help w/component reusability

Reply:
Yeah! I just copied these straight from the existing css but yes, we should clean up.